cadf/dispatcher.sls

```diff
@@ -23,6 +23,16 @@
     - require:
         - file: {{ config.config_dir }}
 
+/etc/logrotate.d/cadf_dispatcher:
```

Comment:
Don'd deploy logrotate config this way, use this approach:
https://github.com/salt-formulas/salt-formula-linux/blob/master/linux/meta/logrotate.yml

Reply:
I have verified, meta option is used for generic purpose module. here we want to use it for specific service and if we go with metadata option, there will be too many changes in other repo as well. I have also found there are specific logrotate policy written for kubernetes, apaches etc. So please consider this code